docs/002-basics-of-ci-with-actions.md

```diff
@@ -2,48 +2,48 @@
 
 ## 1 - Inspect the repository
 
-This repository actually contains a [React](https://reactjs.org/)-application built with [vite](https://vitejs.dev/) that we want to automatically test and build in this part of the workshop.
+This repository contains a [React](https://reactjs.org/)-based application built with [Vite](https://vitejs.dev/). We aim to automate its testing and building in this segment of the workshop.
```

Comment:
I don't like `segment
```suggestion
This repository contains a [React](https://reactjs.org/)-based application built with [Vite](https://vitejs.dev/). We aim to automate its testing and building in this lab.
```

I don't like `segment` for a wording here, an throughout the other parts, we always called it a `lab`, so I'd stay consistent here 🙂